File: docs/components_page/components/dropdown/content.R
library(dashBootstrapComponents)

dropdown <- dbcDropdownMenu(
  list(
    dbcDropdownMenuItem("Header", header = TRUE),
    dbcDropdownMenuItem("An item"),
    dbcDropdownMenuItem(divider = TRUE),
    dbcDropdownMenuItem("Active and disabled", header = TRUE),
    dbcDropdownMenuItem("Active item", active = TRUE),
    dbcDropdownMenuItem("Disabled item", disabled = TRUE),
    dbcDropdownMenuItem(divider = TRUE),
    p(
      "You can have other content here like text if you like.",
      className = "text-muted px-4 mt-4"
    )
  ),
  label = "Menu"
)

File: docs/components_page/components/dropdown/content.jl
using DashBootstrapComponents

dropdown = dbc_dropdownmenu(
    [
        dbc_dropdownmenuitem("Header", header = true),
        dbc_dropdownmenuitem("An item"),
        dbc_dropdownmenuitem(divider = true),
        dbc_dropdownmenuitem("Active and disabled", header = true),
        dbc_dropdownmenuitem("Active item", active = true),
        dbc_dropdownmenuitem("Disabled item", disabled = true),
        dbc_dropdownmenuitem(divider = true),
        html_p(
            "You can have other content here like text if you like.",
            className = "text-muted px-4 mt-4",
        ),
    ],
    label = "Menu",
);
